### PR TITLE
Reword sentence

### PR DESCRIPTION
**Pull request by shaniR**

### CODE REVIEW

File: en/docs/develop-components/develop-a-web-application.md [hunk 1]
```diff
@@ -191,6 +191,6 @@ The following limitations are specific to the Choreo Cloud Data Plane:
 
     If you encounter a blank page or a 502 error after deploying your web application, it typically indicates that the wrong directory is being served. To resolve this issue, please follow these steps:
```
Comment (Nashaath) [inline en/docs/develop-components/develop-a-web-application.md:192]: Shall we remove the word `please` here?

File: en/docs/develop-components/develop-a-web-application.md [hunk 2]
```diff
@@ -191,6 +191,6 @@ The following limitations are specific to the Choreo Cloud Data Plane:
...
 
-    - Double-check the build output directory, especially if you are using a Dockerfile-less preset. Ensure that the specified output directory matches the actual output directory generated during the build process. For example, make sure you enter public/ as the output directory if it is indeed public/ and not build/.
+    - Double-check the build output directory, especially if you are using a Dockerfile-less preset. Ensure that the specified output directory matches the actual output directory generated during the build process. For example, if by mistake you have entered `public/` as the output directory when it should have been `build/`.
```
Comment (Nashaath) [inline en/docs/develop-components/develop-a-web-application.md:194]: Shall we avoid the phrase `by mistake` and perhaps use a term like `erroneously` or so?

**Review by Nashaath — COMMENTED** [review]
Added two minor suggestions.

File: en/docs/develop-components/develop-a-web-application.md [hunk 3]
```diff
@@ -189,8 +189,8 @@ The following limitations are specific to the Choreo Cloud Data Plane:
 
 - **I’m seeing a blank page or a 502 error after I deploy my web application**
 
-    If you encounter a blank page or a 502 error after deploying your web application, it typically indicates that the wrong directory is being served. To resolve this issue, please follow these steps:
+    If you encounter a blank page or a 502 error after deploying your web application, it typically indicates that the wrong directory is being served. To resolve this issue, follow these steps:
 
-    - Double-check the build output directory, especially if you are using a Dockerfile-less preset. Ensure that the specified output directory matches the actual output directory generated during the build process. For example, make sure you enter public/ as the output directory if it is indeed public/ and not build/.
+    - Double-check the build output directory, especially if you are using a Dockerfile-less preset. Ensure that the specified output directory matches the actual output directory generated during the build process. For example, erronously if you have entered `public/` as the output directory when it should have been `build/`.
```
Comment (Nashaath) [inline en/docs/develop-components/develop-a-web-application.md:194]:
```suggestion
    - Double-check the build output directory, especially if you are using a Dockerfile-less preset. Ensure that the specified output directory matches the actual output directory generated during the build process. For example, if you have erroneously entered `public/` as the output directory when it should have been `build/`.
```

Comment (shaniR) [reply en/docs/develop-components/develop-a-web-application.md]: fixed

**Review by Nashaath — APPROVED** [review]

**Pull request merged**